### ISSUE TITLE
Latest update not added to flatpak after more than 24 hours

### ISSUE THREAD

**Issue by Arcaly** [flathub/sh.ppy.osu#82]
Not sure if this is intended, however after checking the GitHub page it seems it isn't.

On the start screen you get the notification "A newer release of osu! has been found (2022.1101.0 → 2022.1117.0).". The update has been added yesterday IIRC and normally it enters the flatpak version in a couple of hours. Now over 24 hours later, still no sign of the update being added. 

I went into the flathub page to check for any news and to the GitHub page to check if there was any change, but for some reason the flatpak integrator bot hasn't made any commits like usual. What I want to know is if this is a bug or an intentional delay and how long it will be until it is resolved.

**Comment by peppy** [comment]
See https://github.com/flathub/sh.ppy.osu/pull/81

**Comment by Arcaly** [comment]
Oh I see.

TL;DR Is flathub still hasn't added support for desktop entries version 1.5 so the bot fails to build the package. It's not an issue on the osu! end so I'll be closing the issue.
